clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndex.scala

```diff
@@ -90,27 +89,39 @@ object ElasticsearchIndex extends ModelAutoDerivation {
   lazy val dateTimeFormatter: DateTimeFormatter =
     DateTimeFormatter.ofPattern("yyyy/MM/dd")
 
-  val WgsUbam: ElasticsearchIndex[WgsUbamIndex.type] =
+  val Ubam: ElasticsearchIndex[UbamIndex.type] =
     new ElasticsearchIndex(
-      WgsUbamIndex,
+      UbamIndex,
+      // We need to keep this name consistent with GCS, so we cannot easily change it.
+      // Since we compute GCS paths from the ES index name, inconsistency would break GCS paths.
+      "wgs-ubam",
```

Comment:
These were named constants before. If you're going to pull them out of `SemiAutoClioIndex`, I'd keep them as named constants here.